### DISCUSSION TITLE
<< Where to start? >>

### DISCUSSION THREAD

**Discussion by icamps** [SakanaAI/AI-Scientist#73] [Q&A]
Hi.

I read the docs here and the article, but I am a little lost about how I can actually setup AI-Scientist to start writing in a given topic like, for example, astrology.

I need to manually add my ideas for the paper inside `seed_ideas.json`? 

Thanks in advance.

PS: a simple tutorial will be of great help for beginners.

**Comment by Ramzee-S** [comment]
Hi, i am having the same issue. I went trough the paper and the code in the repo. To me it is not really clear what was specified for the example papers. It seems for the data templates and prompts for each example paper were carefully prepared. And it does not seem so easily transferable to new ideas. But maybe i am just misunderstanding. 

Did you figure out something more? 
Did anyone get this code to run and generate a new manuscript on a user-seeded topic? 

Any help would be appreciated.
